plugins/repository-azure/src/main/java/org/elasticsearch/cloud/azure/storage/AzureStorageSettings.java

```diff
@@ -51,6 +51,12 @@
         key -> SecureSetting.secureString(key, null));
 
     /**
+     * Azure endpoint suffix. Default to core.windows.net (CloudStorageAccount.DEFAULT_DNS).
+     */
+    public static final Setting<String> ENDPOINT_SUFFIX_SETTING = Setting.affixKeySetting(PREFIX, "endpoint_suffix",
+        key -> SecureSetting.simpleString(key));
```

Comment:
I don't think the endpoint needs to be a secure setting.

Reply:
Yes, of course. Thanks.